tools/track_participation.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Filename: track_participation.py
+
+import sys
+import os
+import logging
+import json
+from github import Github, GithubException
+import getopt
+from datetime import datetime, timedelta
+from zoneinfo import ZoneInfo
+from prettytable import PrettyTable
+
+PRINT_IN_MARKDOWN = False
+PUBLISH = False
+
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+REPO_FULLNAME = os.getenv("REPO_FULLNAME")
+ISSUE_NUMBER = os.getenv("TRACKER_ISSUE_NUMBER")
+
+config = json.load(open("tools/track_participation_config.json"))
```

Comment:
```suggestion
config = json.load(open("track_participation_config.json"))
```

I think this should be sufficient?

Reply:
It will work when running locally from the tools folder, but the workflow will not find the file as it runs Python in the repo's root dir. 

Could change that in the workflow file, but I thought this was cleaner.

Reply:
> Could change that in the workflow file, but I thought this was cleaner.

I think otherwise because the this path is only true with respect to the CI job so it should be in workflow file only.